korman/properties/modifiers/gui.py

```diff
@@ -427,6 +427,14 @@ class PlasmaLinkingBookModifier(PlasmaModifierProperties, PlasmaModifierLogicWiz
                                 description="The object the avatar will click on to activate the Linking Book GUI",
                                 type=bpy.types.Object,
                                 poll=idprops.poll_mesh_objects)
+    shareable = BoolProperty(name="Shareable",
+                             description="Enable the Book to be Shareable (MOUL private instance only)",
+                             default=False,
+                             options=set())
+    shr_region = PointerProperty(name="Share Region (optional)",
```

Comment:
I think this should be `share_region`. I think I would really have trouble remembering this abbreviation ;)

Reply:
Agreed. Only issue I see is it'll overlap with the `share_region` definition of the new node (line 647). I can rename that share_region_msb to differentiate.

Reply:
Having the same name would be a good thing, IMO 😄

Reply:
Heh, OK. I thought it'd cause problems, but I defer to your expertise as always. 😃